WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,4 +1,7 @@
-- Font size and line height are now supported for block themes.
-- Add new stat cards in the Settings menu on the Stats screen.
-- When changing the media in an Image block that’s also a post’s Featured Image, get a prompt to replace the Featured Image.
-- Share text, images, and video in the app using your device’s Share action.
+* [*] Comment editor: Notification comments support in the new editor [https://github.com/wordpress-mobile/WordPress-Android/pull/15884]
+* [*] Site creation: Fixed bug where sites created within the app were not given the correct time zone, leading to post scheduling issues. [https://github.com/wordpress-mobile/WordPress-Android/pull/15904]
+* [*] Login Epilogue: Fixed bug where if no sites available, then new login epilogue screen without clickable sites and with "create new site" is shown instead of no sites empty "My Site" view. [https://github.com/wordpress-mobile/WordPress-Android/pull/15944]
```

Comment:
Not entirely sure that the Login Epilogue is the same for WP and JP, and if not how this entry should look like for Jetpack (or if it should be there at all)

Reply:
Same as https://github.com/wordpress-mobile/WordPress-Android/pull/15986#discussion_r811115368 : should we still keep that in the JP release notes anyway even if the feature is now enabled, given that users of Jetpack 19.2 never saw that Login Epilogue before (#17914 not being in 19.2), so telling them that a bug was fixed on something that will be new to them feels strange?

Reply:
> given that users of Jetpack 19.2 never saw that Login Epilogue before 

Makes sense actually. 😅 We can remove this entry for JP.

Reply:
Addressed in [681333f](https://github.com/wordpress-mobile/WordPress-Android/pull/15986/commits/681333ff5bee95e216234470184fd4363154d507)